examples/src/parsers.rs

```diff
@@ -0,0 +1,146 @@
+//! This file contains a collection of simple Enums which allow a nice command line interface.
+//!
+//! For each enum, we allow the user to specify the enum either using the whole string or any substring
+//! which fully determines the choice. We additionally add a few extra aliases if other natural ones exist.
+//!
+//! For most of the enums, this allows the user to
+
+use clap::builder::PossibleValue;
+use clap::ValueEnum;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum FieldOptions {
+    BabyBear,
+    KoalaBear,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum ProofObjectives {
+    Blake3Permutations,
+    KeccakFPermutations,
+    Poseidon2Permutations,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum DFTOptions {
```

Comment:
nit: the naming [guideline](https://rust-lang.github.io/api-guidelines/naming.html) prefers `Dft`